ovn-tester/ovn_utils.py

```diff
@@ -912,3 +912,8 @@ def uuid_transaction(self, func):
     def ts_add(self):
         log.info('Creating transit switch')
         self.uuid_transaction(partial(self.idl.ts_add, 'ts'))
+
+
+def distribute_n_tasks_per_clusters(n_tasks, n_clusters):
+    div, rest = divmod(n_tasks, n_clusters)
+    return [div + 1 if i < rest else 0 for i in range(n_clusters)]
```

Comment:
@dceara I think this is not correct, e.g: 
for div = 2, rest = 3, it will end up in [3, 3, 3, 0, 0, 0].

I think it should be something like:
return [div + 1 if i < rest else div for i in range(n_clusters)]

Reply:
Good catch, the precedence of operations in Python got me.  I'll fix this.